scripts/externalTests/common.sh

```diff
@@ -572,6 +572,13 @@ function gas_report_path
     echo "${DIR}/gas-report-${preset}.rst"
 }
 
+function compilation_time_report_path
+{
+    local preset="$1"
```

Comment:
Maybe we should have something like:
```sh
    (( $# == 1 )) || assertFail
```
Just to ensure that `preset` will have some value.

Reply:
Oh, just noticed that we do not do that in other places as well, for instance, `gas_report_path`.

Reply:
Sure why not.

Yeah, we don't always do this and it's generally a pain in the ass to have to do things like this for every function in Bash. But it also does not hurt.

Reply:
While at it I also adjusted it to allow more than one argument in the command. It's meant to be used primarily with those functions from external tests and they don't take arguments but a more general form might be handy for other uses.

Reply:
But you didn't adjust the `compilation_time_report_path`. Or maybe I misunderstood your comment haha
I'm fine with the changes anyway ;)

Reply:
Ah, sorry. I didn't look at the suggestion closely enough and I thought you were talking about the other function :)

Anyway, I just merged the PR and I also think it's good enough without this - doing this for every function would just be annoying. It's something I'd really like Bash to do automatically, but we can't have nice things :)